packages/twitch-helpers/src/index.ts

```diff
@@ -0,0 +1,27 @@
+export * from './lib/constants';
+export * from './lib/enums';
+export * from './lib/twitch';
+export type {
+	TwitchEventSubEvent,
+	TwitchEventSubOnlineEvent,
+	TwitchEventSubResult,
+	TwitchEventSubVerificationMessage,
+	TwitchFetchUsersParameters,
+	TwitchHelixBearerToken,
+	TwitchHelixGameSearchResult,
+	TwitchHelixOauth2Result,
+	TwitchHelixResponse,
+	TwitchHelixStreamsResult,
+	TwitchHelixUserFollowsResult,
+	TwitchHelixUsersSearchResult,
+	TwitchOnlineEmbedData
+} from './lib/types';
```

Comment:
*Not a request*. `export type *` would be a wonderful feature to have.

TypeScript, when??